homeassistant/components/climate/mill.py

```diff
@@ -117,6 +117,7 @@ def device_state_attributes(self):
             "open_window": self._heater.open_window,
             "heating": self._heater.is_heating,
             "controlled_by_tibber": self._heater.tibber_control,
+            "gen_1_heater": self._heater.is_gen1,
```

Comment:
Would it make sense to have something like `heater_generation` that is an integer?

Reply:
Fixed